### PR TITLE
Developer Doc: REST API 0.11, 0.12, 0.13 update

### PR DESCRIPTION
**Pull request by Mirobit**
The REST API developer documentation as of 0.13 (no changes in 0.13.1). 

I did not fully understand the GetUtxos call:
- How are _Outpoints_ defined? What is <n>?
- What is the point of the return field _bitmap_?

### CODE REVIEW

**Comment by Mirobit** [comment]
@harding Thanks for the review and for expanding the GetUtxos call. I am glad that someone is doing this.

File: _includes/devdoc/bitcoin-core/rest/intro.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ an **unauthenticated** HTTP REST interface.  The interface runs on the
 same port as the JSON-RPC interface, by default port 8332 for mainnet and
 port 18332 for testnet. It must be enabled by either starting Bitcoin
 Core with the `-rest` option or by specifying `rest=1` in the
-configuration file.
+configuration file. Make sure that the RPC interface is also activated.
+Set `server=1` in `bitcoin.conf` or supply the `-server` argument when 
+starting the program.
```
Comment (Mirobit) [inline _includes/devdoc/bitcoin-core/rest/intro.md]: Thanks, you are right. Addressed this in the latest commit.

**Comment by wbnns** [comment]
@Mirobit Thanks. Could you please resolve the conflicts? Once you do I'll move to #1479. 👍

**Comment by Mirobit** [comment]
@wbnns Thanks, rebased.

**Comment by wbnns** [comment]
@Mirobit Thanks!

Unless others object, this will be merged on Sunday, February 12th.

**Pull request merged**